src/main/common/filter.h

```diff
@@ -55,12 +55,32 @@ typedef struct ptnFilter_s {
     uint8_t order;
 } ptnFilter_t;
 
+//Max N = 25
+typedef struct {
+    int windowSize;
+    int windowBufIndex;
+    int N;
+    float buf[51];
+    float minBufA[51];
+    float luluInterimA[51];
+    float luluInterimB[51];
+} luluFilter_t;
+
+typedef struct {
+	luluFilter_t A;
+	luluFilter_t B;
+} luluFilter2_t;
+
 typedef enum {
     FILTER_PT1 = 0,
     FILTER_BIQUAD,
     FILTER_PT2,
     FILTER_PT3,
-    FILTER_PT4,
```

Comment:
the trailing comma is used just for these situations, to make an addition to not cause a change in an already existing line.
You should leave it there (and put one on your entry too)

Reply:
I'll fix this up, thank you and appreciated!